tasks.py

```diff
@@ -918,7 +918,7 @@ def _get_marker_dependencies(markers: str | Sequence[str]) -> list[TestDependenc
     iterable=["markers", "requirements_dev"],
     help={
         "markers": "Optional marker to install dependencies for. Can be specified multiple times.",
-        "requirements_dev": "Short name of `requirements-dev-*.txt` file to install, e.g. test, spark, cloud etc. Can be specified multiple times.",  # noqa: E501
```

Comment:
The cloud requirements file still exists so you can leave that example in here (though, please add a comma if you do) and below.

Reply:
updated, ty